configuration/configure-storage-management.md

```diff
@@ -0,0 +1,33 @@
+---
+description:  This article describes how you can set up Storage Management in Syskit Point. 
+---
+
+# Storage Management
+
+The Syskit Point Storage Management feature helps you keep your environment clutter-free, optimize the usage of your storage, and gain insight into how your storage is distributed across your tenant.
+
+For more information on this feature, [take a look at the Optimize Storage article](../governance-and-automation/optimize-storage.md). 
+
+To set up the Storage Management feature within Syskit Point, complete the following steps:
+
+  * **Go to the Settings (1)** of your Syskit Point
+  * Under the General section, **click the Storage Management button (2)**
+  * **Click the Collect storage data checkbox (3)** to turn on the storage management feature
```

Comment:
ovo provjeriti s Matejem. mislim da je by default uključeno tako da ne treba to podešavati.

Reply:
isto tako, potrebno napisati što je posljedica uključivanja/isključivanja ove opcije